packages/d/dotnet/package.yml

```diff
@@ -0,0 +1,82 @@
+name       : dotnet
+version    : 8.0.1
+release    : 1
+source     :
+    - git|https://github.com/dotnet/dotnet.git : b27976e5a6850466ee5b4ce24f91ee93bef645f7
+homepage   : https://dot.net/core
+license    : MIT
+component  : programming.dotnet
+summary    : .NET is a free, open-source, cross-platform framework for building modern apps and cloud services.
```

Comment:
Summary should be shorter.

Also, you should add a different summary and description for the subpackages to describe what they are and what they need.